storage/storagepb/kv.proto

```diff
@@ -14,11 +14,14 @@ message KeyValue {
   int64 create_revision = 2;
   // mod_revision is the last modified revision of the key.
   int64 mod_revision = 3;
+  // tombstone_revision is the tombstone revision of the key.
+  // If the key is not tombstoned, its revision is set to zero.
+  int64 tombstone_revision = 4;
```

Comment:
> It belongs to key-value struct IMO.

We add tom_rev just for watch layer. So basically this provides the functionality for watch layer. If we do not have the watch layer, then we do not need to do it at all, right? 

Then why does this functionality belong to kv layer?